src/table/table.scss

```diff
@@ -367,6 +411,43 @@
     flex-basis: $iui-l * 2;
   }
 
+  &.iui-sticky {
+    position: sticky;
+    z-index: 1;
+  }
+
+  .iui-shadow-right,
+  .iui-shadow-left {
+    position: absolute;
+    top: -1px;
+    bottom: -1px;
+    width: 30px;
+    transition: box-shadow 0.3s;
```

Comment:
This doesn't seem to do anything because the `<div class="iui-shadow-right">` is removed from the DOM.  Transitions would be nice, but if it's too tricky to get working it's fine as is.

Reply:
Good catch! Removed